### PR TITLE
Added a docs file for mbmbsmackend cr file.

### PR DESCRIPTION
**Pull request by AdamSaleh**
First attempt at the docs-file.

Might need few additional entries for the deployment part of the CR.

### CODE REVIEW

File: docs/source/user-guide/mbs-backend.rst [hunk 1]
```diff
@@ -0,0 +1,238 @@
...
+
+`Mbox Module Build Service Backend Custom Resource Definition (CRD) <https://raw.githubusercontent.com/fedora-infra/mbbox/master/mbox-operator/deploy/crds/apps.fedoraproject.org_mbmbsbackends_crd.yaml>`_
+
+Parameters
+==========
+
++-------------------------+-----------------------------------------------+---------+
+| Name                    | Default Value                                 | Type    |
++=========================+===============================================+=========+
+| image                   | quay.io/fedora/koji-builder:latest            | string  |
```
Comment (odra) [inline docs/source/user-guide/mbs-backend.rst:23]: The correct default image value is `quay.io/fedora/mbs-backend:latest/`

File: docs/source/user-guide/mbs-backend.rst [hunk 2]
```diff
@@ -0,0 +1,245 @@
...
++-------------------------+-----------------------------------------------+---------+
+| Name                    | Default Value                                 | Type    |
++=========================+===============================================+=========+
+| image                   | quay.io/fedora/mbs-backend:latest             | string  |
++-------------------------+-----------------------------------------------+---------+
+| replicas                | 1                                             | int     |
++-------------------------+-----------------------------------------------+---------+
+| configmap               | mbs-backend-configmap                         | string  |
++-------------------------+-----------------------------------------------+---------+
+| shared_pvc              | koji-hukoji-hub-mnt-pvcb-mnt-pvc                              | string  |
```
Comment (odra) [inline docs/source/user-guide/mbs-backend.rst:29]: Misspelling "koji-hukoji-hub-mnt-pvcb-mnt-pvc", should be "koji-hub-mnt-pvc"

**Comment by odra** [comment]
just one comment, looks good otherwise.

**Comment by odra** [comment]
recheck

**Comment by AdamSaleh** [comment]
recheck

**Comment by AdamSaleh** [comment]
recheck

**Comment by AdamSaleh** [comment]
Damn it, the tests are kinda flaky :-/

**Comment by odra** [comment]
recheck

**Comment by odra** [comment]
I will merge it since the docs tests are passing.

**Pull request merged**